Sources/VLCPlaybackController.m

```diff
@@ -793,12 +784,17 @@ - (void)pause
     [[NSNotificationCenter defaultCenter] postNotificationName:VLCPlaybackControllerPlaybackDidPause object:self];
 }
 
+- (void)setShuffleMode:(BOOL)shuffleMode
+{
+    _shuffleMode = _pbcSettings.shuffleMode = shuffleMode;
```

Comment:
Do we still need to keep a copy the different playback setting properties? Could we expose `PlaybackSettings` directly?

Reply:
Ah you mean we should remove those settings entirely from the playbackcontroller ? yes I like that idea

Reply:
You need to set this information on the _listplayer or _mediaplayer. The settings are just holding that information. I would need to have a settingsDidChange method.